### DISCUSSION TITLE
appendToFile

### DISCUSSION THREAD

**Discussion by jjcpa** [rfennell/AzurePipelines#1147] [Q&A] [answered]
Hi, appendToFile to append the new release note to previous ones. Is there a way to do this in reverse order? Keeping the latest release note on the top.

Thanks
Jjcpa

**Comment by rfennell** [comment, marked as answer]
Yes, as detailed in this project [wiki](https://github.com/rfennell/AzurePipelines/wiki/GenerateReleaseNotes---Node-based-Cross-Platform-Task)

Parameter | Description
-- | --
ReplaceFile | If this is set the output overwrites and file already present.
AppendToFile | If this is set, and replace file is false then then output is appended to the output file. If false it is preprended.

As the documentation says you need
- replaceFile - false
- appendtoFile - false
